application/forms/Config/UserBackendConfigForm.php

```diff
@@ -45,6 +52,12 @@ public function init()
         $this->setName('form_config_authbackend');
         $this->setSubmitLabel($this->translate('Save Changes'));
         $this->setValidatePartial(true);
+        $this->setCustomBackend();
+    }
+
+    protected function setCustomBackend()
+    {
+        $this->customBackends = UserBackend::getCustomBackendConfigForms();
```

Comment:
Aaah, my eyes bleed :sob: :joy:

Do I understand this correct, that you did that just to avoid the error you otherwise get in the test you've patched as well?

This is bad. Designing code in such a way, that it's testable is never a good thing. Do that without this *setter* directly in the init method.

I've added a suggestion below how to make the test working.

Reply:
Yes you're totally right, let me fix this.